Sources/SWBCore/Specs/Tools/SwiftCompiler.swift

```diff
@@ -2127,6 +2127,11 @@ public final class SwiftCompilerSpec : CompilerSpec, SpecIdentifierType, SwiftDi
                 return (inputs, outputs)
             }()
 
+            if cbc.scope.evaluate(BuiltinMacros.PLATFORM_REQUIRES_SWIFT_MODULEWRAP) && cbc.scope.evaluate(BuiltinMacros.GCC_GENERATE_DEBUGGING_SYMBOLS) {
+                let moduleWrapOutput = Path(moduleFilePath.withoutSuffix + ".o")
```

Comment:
Windows really should use `.obj` instead of `.o`.

Reply:
I filed https://github.com/swiftlang/swift-build/issues/149 to track this because it needs to be done in a few other places at the same time to avoid a regression

Reply:
@compnerd I think it's more like "MSVC ABI should use .obj and gcc/mingw ABI should use .o", right?